css-canvas-medical/styles.css

```diff
@@ -0,0 +1,197 @@
+#privacy {
+  margin-right: 40px;
+}
+
+#end {
+  padding: 100px 10%;
+}
+
+.indented {
+  padding-left: 10%;
+  padding-right: 10%;
+}
```

Comment:
In utility libraries this may be called something like `px-10` which would say that the padding on the x axis is 10 (something but usually in `rem`).

Just a note for you if you want to try naming things similarly in the future.